.github/workflows/build_dev_documentation.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           repository: 'huggingface/doc-builder'
           path: doc-builder
+          ref: "kit"
```

Comment:
Note: remove after PR https://github.com/huggingface/doc-builder/pull/94 is merged